homeassistant/helpers/template.py

```diff
@@ -2178,45 +2198,62 @@ def _render_with_context(
         return template.render(**kwargs)
 
 
-class LoggingUndefined(jinja2.Undefined):
+def make_logging_undefined(
```

Comment:
Why not add an `__init__` to the class ?

Reply:
Because we're not returning an instance of the class, we're returning the class itself.